test/test_google_ads_client.rb

```diff
@@ -33,8 +33,27 @@ class GoogleAdsClient
       end
     end
   end
+
+  # We don't actually want to run any auth code, just test the setup.
+  module Auth
```

Comment:
I feel very strongly that you shouldn't patch classes for the *entire execution of anything* once the tests are loaded, let's talk about how we can do this patch more safely.